tests/components/homepluscontrol/test_options_flow.py

```diff
@@ -0,0 +1,84 @@
+"""Test the Legrand Home+ Control config options flow."""
```

Comment:
Move these tests to the `test_config_flow.py` module.

Reply:
Merged these into the test_config_flow module and remove test_options_flow.